frontend/src/pages/projects/screens/spawner/storage/CreateNewStorageSection.tsx

```diff
@@ -31,8 +31,11 @@ const CreateNewStorageSection = <D extends CreatingStorageObject>({
         <NameDescriptionField
```

Comment:
It would be great to replace this with `K8sNameDescriptionField` to allow the user to manually set the K8s resource name if desired and would work towards [RHOAIENG-14746](https://issues.redhat.com/browse/RHOAIENG-14746)

Reply:
There's a lot in-flight with this PR as it is right now, and since the replacement will involve some rework of this PR, I would prefer it be a separate issue if not just a part of the JIRA you've cited here.